holmes/core/toolset_manager.py

```
@@ -15,430 +21,443 @@
+        # Different content = stale
+        if data.get("_content_hash") != content_hash:
+            return True
+
+        # Too old = stale (for non-file toolsets)
+        age = time.time() - data.get("_timestamp", 0)
+        if age > max_age_seconds:
+            return True
+
+        # Check if any toolset has invalid/unknown status
+        toolsets = data.get("toolsets", {})
+        for name, toolset_data in toolsets.items():
```

Ruff (0.12.2)

Loop control variable `name` not used within loop body

Rename unused `name` to `_name`

(B007)